ui/redux/reducers/stripe.js

```diff
@@ -15,7 +16,13 @@ const defaultState: StripeState = {
   customerSetupResponse: undefined,
 };
 
-reducers[ACTIONS.SET_ACCOUNT_STATUS] = (state, action) => ({ ...state, accountStatus: action.data });
+reducers[ACTIONS.STRIPE_ACCOUNT_STATUS_START] = (state, action) => ({ ...state, accountStatusFetching: true });
+reducers[ACTIONS.STRIPE_ACCOUNT_STATUS_COMPLETE] = (state, action) => ({
+  ...state,
+  accountStatusFetching: false,
+  accountStatus: action.data,
+});
```

Comment:
If you block `account/status` to emulate network or api failure, the button becomes:

<img width="467" alt="image" src="https://user-images.githubusercontent.com/64950861/197172213-d3cb53b9-f85a-4446-803f-6c77533a8aed.png">

Reply:
:small_blue_diamond: Just a very minor corner-case that forces the user to refresh.  Can choose not to handle.

- An error in the fetch will always clear out the redux data.  It's generally fine because most clients check if it was already fetched or not, except for `WalletFiatBalance` which perhaps needed to do it every time.  So, if you enter `/$/wallet` and a glitch happen, you're stuck in the "no account" state.